ee/cis/win-10/cis-policy-queries.yml

```diff
@@ -931,4 +931,154 @@ spec:
   tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_2.3.10.12
   contributors: rachelelysia
 ---
-
+apiVersion: v1
+kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Windows Firewall: Private: Firewall state' is set to 'On (recommended)' (Automated)
```

Comment:
I think we do not add the (Automated) or (Manual) distinction here

Reply:
@marcosd4h why is that?